docs/client/session.md

```diff
@@ -0,0 +1,25 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
+
+The primary purpose of sessions is to provide insight into the series of user actions or events that lead a critical or 
+interesting occurrence in the application's use. Sessions also provide a means to quantify overall application usage.
+
+
+## Attributes
+
+This attribute can be applied to all signals generated by an agent. It may be omitted from metrics, due to their aggregated nature. 
+
+| Attribute   | Type    | Description | Examples                             | Requirement level | 
+|-------------|---------|-------------|--------------------------------------|-------------------|
+| session.id  |  String | A GUID      | 00112233-4455-6677-8899-aabbccddeeff | required          |
+
+## How a session operates
+After a period of timeout, the `session.id` MUST be refreshed.
+The timeout period MUST be restarted when any Logs, Events, or Spans are recorded.
+The session timeout period CAN be customized.
+Default session timeout SHOULD be 30 minutes. This should only be done when the agent is configured, and MUST NOT be updated in the middle of a session.
+
+Maximum session length SHOULD NOT exceed 4 hours.
```

Comment:
Session would be an SDK construct and would be configured when SDK is setup.  Hence the session configuration parameters could only be changed if SDK is re-initialized.

Session timeout and max session duration are both configurable, with some defaults. Could you please add these 2 in a table so they are highlighted as configurable parameters?

Reply:
Certainly! I'll try to find an example of how this should look.